XamlControlsGallery/DataModel/ControlInfoData.json

```diff
@@ -1943,6 +1943,71 @@
           "RelatedControls": [
           ]
         },
+        {
+          "UniqueId": "AnimatedIcon",
+          "Title": "AnimatedIcon",
+          "Subtitle": "An element that displays and controls an icon that animates when the user interacts with the control.",
+          "ImagePath": "ms-appx:///Assets/Animations.png",
+          "Description": "An element that displays and controls an icon that animates when the user interacts with the control.",
+          "Content": "<p>Look at the <i>AnimatedIconPage.xaml</i> and <i>AnimatedIconPage.xaml.cs</i> files in Visual Studio to see the full code for this page.</p>",
+          "IsNew": true,
+          "Docs": [
+            {
+              "Title": "AnimatedIcon - API",
+              "Uri": "https://www.docs.microsoft.com/uwp/api/microsoft.ui.xaml.controls.animatedicon"
+            },
+            {
+              "Title": "AnimatedIcon - Guidance ",
+              "Uri": "https://docs.microsoft.com/windows/uwp/design/style/animated-icon"
+            },
+            {
+              "Title": "Lottie Overview",
+              "Uri": "https://docs.microsoft.com/windows/communitytoolkit/animations/lottie"
+            },
+            {
+              "Title": "Lottie Windows - GitHub",
+              "Uri": "https://www.docs.microsoft.com/uwp/api/microsoft.ui.xaml.controls.animatedvisualplayer"
+            }
+          ]
+        },
+        {
+          "UniqueId": "IconControls",
+          "Title": "IconControls",
```

Comment:
Would recommend `IconElement` as that's the base class.

Reply:
That's a better title. I couldn't think of another name that makes it clear that there's more than one control on the sample page.